tests/unit/utils/test_utils_pubnote.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Unit tests for pubnote parsing."""
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.utils.pubnote import build_pubnote
+
+
+def test_build_pubnote():
+    unicode_title = u'φοο_title'
+    unicode_volume = u'φοο_volume'
+
+    expected = u'{},{}'.format(unicode_title, unicode_volume)
```

Comment:
Mh, read this: https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html